wiki/Tournaments/Official_support/de.md

```diff
@@ -102,14 +111,17 @@ In Bezug auf die Rolle des Personals gelten folgende Regeln:
+- Alle Mitarbeiter, die in irgendeiner Weise an der Durchführung eines Turniers beteiligt sind, dürfen zum jeweiligen Zeitpunkt nicht gesperrt sein.
 
 Von Benutzern mit einer aktiven Turniersperre wird erwartet, dass sie den Status ihrer Turniersperre bei allen offiziell unterstützten Turnieren offenlegen, bei denen sie helfen wollen. Sie dürfen sich nach dem Ermessen des Veranstalters als Streamer, Kommentatoren oder Grafikdesigner beteiligen, aber wir ermutigen zu sorgfältiger Abwägung von solchen Entscheidungen, da die genannten Personen bereits einmal gegen die Regeln verstoßen haben (oder mehrmals). Sie dürfen keine andere Mitarbeiterrolle ausüben, ES SEI DENN, der Veranstalter bittet beim Erstantrag auf Unterstützung um eine Ausnahme für sie. Das Account-Support-Team wird daraufhin die Spieler mit Blick auf deren Geschichte von Fall zu Fall prüfen.
 
-Jeder, dessen Verhalten dazu führt, dass das Turnier seine offizielle Unterstützung verliert, darf mindestens 4 Monate lang nicht dieselbe Rolle in einem anderen Turnier mit Abzeichen spielen oder dessen Veranstalter sein.
+Mitarbeiter, deren Verhalten direkt dazu führt, dass das Turnier seine offizielle Unterstützung verliert, dürfen im Ermessen des Support-Teams mindestens 4 Monate lang nicht dieselbe Rolle in einem anderen Turnier mit Abzeichen übernehmen oder deren Veranstalter sein. Falls diese Auszeit für einen Nutzer gilt, wird dieser direkt vom [Turnier-Team](mailto:tournaments@ppy.sh) informiert.
 
 ### Spieler
 
-Von Teilnehmern in offiziell unterstützten Turnieren wird erwartet, dass sie sich zu jedem Zeitpunkt an die [osu! Community-Regeln](/wiki/Rules) halten, unabhängig von ihrem Rang, Errungenschaften oder andere Leistungen. Von ihnen wird ebenfalls erwartet, dass sie alle Freiwilligen, die an der Ausrichtung eines Turniers beteiligt sind, mit Respekt behandeln und nicht absichtlich versuchen, ihre Zeit zu verschwenden oder anderweitig mit ihren Aktionen die Durchführung eines Turniers übermäßig zu erschweren.
+Von Teilnehmern in offiziell unterstützten Turnieren wird erwartet, dass sie sich zu jedem Zeitpunkt an die [osu! Community-Regeln](/wiki/Rules) halten, unabhängig von ihrem Rang, ihren Errungenschaften oder anderen Leistungen. Von ihnen wird ebenfalls erwartet, dass sie alle Freiwilligen, die an der Ausrichtung eines Turniers beteiligt sind, mit Respekt behandeln und nicht absichtlich versuchen, ihre Zeit zu verschwenden oder anderweitig mit ihren Aktionen die Durchführung eines Turniers übermäßig zu erschweren.
+
+Des Weiteren wird von den Spielern erwartet, dass sie an einem bestimmten Turnier *hinreichend teilgenommen* haben, um sich für etwaige Preise zu qualifizieren — sich einfach als Mitglied eines Siegerteams zu qualifizieren ist nicht ausreichend.
```

Comment:
**hinreichend teilgenommen**: `reasonably` can be translated as "behaved well" and "put in enough effort". This rule specifically refers to people that qualify together with a team for the badge prizes. Receiving badges without playing enough matches is forbidden.

See [this discussion on Discord](https://discord.com/channels/188630481301012481/867773019015610389/1152574080319492158).